jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- job:
+    name: osd-cluster-integreatly-uninstall
+    display-name: 'OSD Cluster Integreatly Uninstall'
+    project-type: pipeline
+    concurrent: false
```

Comment:
Ok for now, however, we should probably create a followup JIRA ticket to allow for concurrent builds once two issues linked in https://issues.jboss.org/browse/INTLY-3254 are finished. If that makes sense I will create the ticket

Reply:
Yes I agree. OK thanks. Feel free to create it.

Reply:
created: https://issues.jboss.org/browse/INTLY-3716